knowledge-base/styling-tooltips-in-html5-based-report-viewers.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Styling Tooltips in HTML5-based Report Viewers 
+description: Showing tooltips in a webpage is a very common task and there are lots of solutions for any magnitude of complexity. In this article, we will demonstrate some ways the Tooltips in HTML5-based Report Viewers.
```

Comment:
Remove the first part of the description here, as it does not relate to keywords and might bring confusion and wrong indexing. For example:
FROM: Showing tooltips in a webpage is a very common task and there are lots of solutions for any magnitude of complexity. In this article, we will demonstrate some ways the Tooltips in HTML5-based Report Viewers.
TO: How to show Tooltops in HTML5 Report Viewer in different ways.